conan/tools/gnu/autotoolsdeps.py

```diff
@@ -17,18 +18,33 @@ def ordered_deps(self):
             self._ordered_deps = [dep for dep in reversed(deps.values())]
         return self._ordered_deps
 
+    def _get_required_deps(self):
```

Comment:
Scared to change the API of ordered_deps, so I created this private method instead of changing it